### DISCUSSION TITLE
How to get the student Pack offer available on github education ?

### DISCUSSION THREAD

**Discussion by jaxkashif34** [skills/.github#34] [Q&A]
Is there anyone here who get the student pack offer please contact me I really need that thanks....

**Comment by deanwhitbread** [comment]
To get the student pack offer, you must add your education email address onto your profile. You can do this by going into Settings > Emails. Underneath the 'Add email address' section, type in your school email address and click the 'Add' button. 

Go back to the student pack offer and ensure you select the student option for individuals. You will need to select your education email address to submit an application for the offer.

**Comment by jaxkashif34** [comment]
ok thanks @deanwhitbread  I already done this but here I'm talking about any kind of merchandise like a bag pack or something like that.
